### PR TITLE
[zh-cn] sync persistent-volume-claim-v1.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 1]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
 
+- **volumeAttributesClassName** (string)
+
+  <!--
+  volumeAttributesClassName may be used to set the VolumeAttributesClass used by this claim. If specified, the CSI driver will create or update the volume with the attributes defined in the corresponding VolumeAttributesClass. This has a different purpose than storageClassName, it can be changed after the claim is created. An empty string value means that no VolumeAttributesClass will be applied to the claim but it's not allowed to reset this field to empty string once it is set. If unspecified and the PersistentVolumeClaim is unbound, the default VolumeAttributesClass will be set by the persistentvolume controller if it exists. If the resource referred to by volumeAttributesClass does not exist, this PersistentVolumeClaim will be set to a Pending state, as reflected by the modifyVolumeStatus field, until such as a resource exists. More info: https://kubernetes.io/docs/concepts/storage/volume-attributes-classes/ (Beta) Using this field requires the VolumeAttributesClass feature gate to be enabled (off by default).
+  -->
+
+  `volumeAttributesClassName` 可用于设置此声明使用的 VolumeAttributesClass。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:260]:
```suggestion
  `volumeAttributesClassName` 可用于设置此申领所使用的 VolumeAttributesClass。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 2]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
+  如果指定，CSI 驱动程序将使用相应 VolumeAttributesClass 中定义的属性创建或更新卷。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:261]:
```suggestion
  如果设置了此字段，CSI 驱动程序将使用相应 VolumeAttributesClass 中定义的属性创建或更新卷。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 3]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
+  这与 `storageClassName` 的用途不同，可以在创建声明后更改。空字符串值表示不会将 VolumeAttributesClass
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:262]:
```suggestion
  与 `storageClassName` 的用途不同，此属性可以在创建申领之后更改。空字符串值表示不会将 VolumeAttributesClass
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 4]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
+  应用于声明，但一旦设置，就不允许将此字段重置为空字符串。如果未指定且 PersistentVolumeClaim 未绑定，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:263]:
```suggestion
  应用于申领，但一旦设置，就不允许将此字段重置为空字符串。如果未指定且 PersistentVolumeClaim 未绑定，
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 5]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
+  则持久卷控制器将设置默认的 VolumeAttributesClass（如果存在）。
+  如果 VolumeAttributesClass 引用的资源不存在，则此 PersistentVolumeClaim 将被设置为 Pending 状态，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:265]:
```suggestion
  如果 VolumeAttributesClass 所引用的资源不存在，则此 PersistentVolumeClaim 将被设置为 Pending 状态，
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 6]
```diff
@@ -280,13 +251,32 @@ dataSourceRef specifies the object from which to populate the volume with data,
...
+  如 `modifyVolumeStatus` 字段所示，直到存在此类资源。更多信息：
+  https://kubernetes.io/zh-cn/docs/concepts/storage/volume-attributes-classes/（Beta）使用此字段需要启用
+  VolumeAttributesClass 特性门控（默认情况下关闭）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:268]:
```suggestion
  https://kubernetes.io/zh-cn/docs/concepts/storage/volume-attributes-classes/
  
  （Beta）使用此字段需要启用 VolumeAttributesClass 特性门控（默认情况下关闭）。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 7]
```diff
@@ -481,7 +477,66 @@ PersistentVolumeClaimStatus 是持久卷申领的当前状态。
...
-    如果它报告 “ResizeStarted”，则意味着正在调整底层持久卷的大小。
+    如果它报告 “Resizing”，则意味着正在调整底层持久卷的大小。
+
+  - **currentVolumeAttributesClassName** (string)
+
+    <!--
+    currentVolumeAttributesClassName is the current name of the VolumeAttributesClass the PVC is using. When unset, there is no VolumeAttributeClass applied to this PersistentVolumeClaim This is a beta field and requires enabling VolumeAttributesClass feature (off by default).
+    -->
+
+    `currentVolumeAttributesClassName` 是 PVC 正在使用的 VolumeAttributesClass 的当前名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:488]:
```suggestion
    `currentVolumeAttributesClassName` 是 PVC 所使用的 VolumeAttributesClass 的当前名称。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 8]
```diff
@@ -481,7 +477,66 @@ PersistentVolumeClaimStatus 是持久卷申领的当前状态。
...
+    如果未设置，则不会将 VolumeAttributeClass 应用于此 PersistentVolumeClaim。
+    这是一个 Beta 级别字段，需要启用 VolumeAttributesClass 功能（默认情况下处于关闭状态）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:490]:
```suggestion
    这是一个 Beta 级别字段，需要启用 VolumeAttributesClass 特性（默认情况下处于关闭状态）。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 9]
```diff
@@ -481,7 +477,66 @@ PersistentVolumeClaimStatus 是持久卷申领的当前状态。
...
+
+  - **modifyVolumeStatus** (ModifyVolumeStatus)
+
+    <!--
+    ModifyVolumeStatus represents the status object of ControllerModifyVolume operation. When this is unset, there is no ModifyVolume operation being attempted. This is a beta field and requires enabling VolumeAttributesClass feature (off by default).
+    -->
+
+    ModifyVolumeStatus 表示 ControllerModifyVolume 操作的状态对象。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:498]:
```suggestion
    `modifyVolumeStatus` 表示 ControllerModifyVolume 操作的状态对象。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 10]
```diff
@@ -481,7 +477,66 @@ PersistentVolumeClaimStatus 是持久卷申领的当前状态。
...
+     - InProgress
+       InProgress indicates that the volume is being modified.
+     - Infeasible
+      Infeasible indicates that the request has been rejected as invalid by the CSI driver. To
+         resolve the error, a valid VolumeAttributesClass needs to be specified.
+    Note: New statuses can be added in the future. Consumers should check for unknown statuses and fail appropriately.
+  -->
+
+  - **modifyVolumeStatus.targetVolumeAttributesClassName** (string)
+
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:526]: 这一行应该放到537行前面

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md [hunk 11]
```diff
@@ -481,7 +477,66 @@ PersistentVolumeClaimStatus 是持久卷申领的当前状态。
...
+    status 是 ControllerModifyVolume 操作的状态。它可以是以下任一状态：
+    - Pending
+      Pending 表示由于未满足要求（例如指定的 VolumeAttributesClass 不存在）而无法修改 PersistentVolumeClaim。
+    - InProgress
+      InProgress 表示卷正在被修改。
+    - Infeasible
+      不可行表示请求已被 CSI 驱动程序拒绝，因为请求无效。要解决此错误，需要指定有效的 VolumeAttributesClass。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md:533]:
```suggestion
      Infeasible 表示请求已被 CSI 驱动程序拒绝，因为请求无效。要解决此错误，需要指定有效的 VolumeAttributesClass。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**